cves/CVE-2020-7471.yml

```diff
@@ -49,7 +49,7 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: It's possible to send malicious code to a server by putting in the proper sequence of characters to bypass input sanitization, letting people get unrestricted access to the servers database.
```

Comment:
Be more explicit in how this exploit is used and how a user gains access via this exploit. Which input(s) can be broken into this way?

Reply:
I agree with the above comment that you should be more explicit on where this malicious code could be injected. Also, "servers" should be "server's".

Reply:
Will do